docs/coding-guidelines/css.md

```diff
@@ -98,52 +98,9 @@ Under the hood, we are using webpack and its `sass-loader`, for compiling the st
 
 ## Media Queries
 
-We don't do device specific breakpoints, we use layout-specific breakpoints that communicate the state of the UI. DO NOT define your own media queries. Utilize the mixins [provided in `_mixins.scss`](https://github.com/Automattic/wp-calypso/blob/master/assets/stylesheets/shared/mixins/_breakpoints.scss). Furthermore, we are pushing for a mobile-first approach to media queries, meaning your default styles should apply to mobile, and desktop should build on top of that. This means that devices with smaller screens have to process less CSS (which makes sense since they are generally less powerful). You should avoid the use of `<` breakpoints like this:
+To avoid code bloat and have a more consistent experience we should try to use the same breakpoints in all SCSS files.
```

Comment:
Nit: we could be more assertive in this sentence if we wanted 😛 

```suggestion
To avoid code bloat and have a more consistent experience we use the same breakpoints in all SCSS files whenever possible.
```